### DISCUSSION TITLE
2 Github Accounts

### DISCUSSION THREAD

**Discussion by PradyumnaA** [community/community#79542] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have 2 github accounts 1 is of my University and another is for personal use.
1) I have generated 2 public and private keys
As a result I am not able to access both of my github accounts.
I need help from the community.
My doubt sounds lame but I cannot help
I tried pushing in git bash
I tried pushing using VSCode editors
Unfortunately Remote explorer allows only 3 explorers
So I request you guys to help in this scenario.
Please help me
Thanks in advance
Pradyumna A Kubear

**Comment by ghostinhershell** [comment]
Hi @PradyumnaA! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
